PhysicsTools/PatAlgos/plugins/PATElectronProducer.cc

```diff
@@ -605,7 +647,15 @@ void PATElectronProducer::produce(edm::Event & iEvent, const edm::EventSetup & i
 	anElectron.setEcalPFClusterIso(-999.);
 	anElectron.setHcalPFClusterIso(-999.);
       }
-
+      if (addPuppiIsolation_) {
+      anElectron.setIsolationPUPPI((*PUPPIIsolation_charged_hadrons)[elePtr], (*PUPPIIsolation_neutral_hadrons)[elePtr], (*PUPPIIsolation_photons)[elePtr]);
+      anElectron.setIsolationPUPPINoLeptons((*PUPPINoLeptonsIsolation_charged_hadrons)[elePtr], (*PUPPINoLeptonsIsolation_neutral_hadrons)[elePtr], (*PUPPINoLeptonsIsolation_photons)[elePtr]);
```

Comment:
@ishvetso 
I would suggest to have some space inside the conditional statement for the indentation.